src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  if (logo) {
+    logo.addEventListener('click', () => {
+      resolve();
+    });
+  } else {
+    reject(new Error('Logo not found on the page'));
+  }
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('error'));
+  }, 3000);
+});
+
+const createMessage = (text, className = 'message') => {
+  const message = document.createElement('div');
+
+  message.classList.add(className);
+  message.textContent = text;
+
+  return message;
+};
```

Comment:
Consider adding some error handling in case the `text` parameter is not provided. This will make your function more robust.